ios/brave-ios/Sources/Brave/Frontend/Settings/Features/ShieldsPrivacy/FilterLists/FilterListsView.swift

```diff
@@ -14,6 +14,42 @@ import SwiftUI
 /// A view showing enabled and disabled community filter lists
 struct FilterListsView: View {
   private static let dateFormatter = RelativeDateTimeFormatter()
+  enum FilterListUpdateStatus: Equatable {
+    case unknown
+    case updating
+    case updated
+
+    var label: String {
+      switch self {
+      case .unknown: Strings.Shields.updateLists
+      case .updated: Strings.Shields.listsUpdated
+      case .updating: Strings.Shields.updatingLists
+      }
+    }
+
+    var braveImageName: String {
+      switch self {
+      case .unknown, .updating: "leo.refresh"
+      case .updated: "leo.check.circle-outline"
+      }
+    }
+
+    var forgroundColor: Color {
+      switch self {
+      case .unknown: Color(.braveBlurpleTint)
+      case .updated: Color(.braveSuccessLabel)
+      case .updating: Color(.braveDisabled)
+      }
+    }
+
+    var isUpdating: Bool {
+      return self == .updating
+    }
+
+    var isUpdated: Bool {
+      return self == .updated
+    }
```

Comment:
These don't seem useful since your enum does not contain any associated values and thus can be compared directly